Header header
string name
Location[] points